packages/d/descope-core-js.json

```
@@ -0,0 +1,28 @@
+{
+  "name": "descope-core-js-sdk",
+  "filename": "descope-core-js.min.js",
+  "description": "Descope JavaScript core SDK",
+  "keywords": ["descope", "descope-core-js-sdk"],
+  "homepage": "https://descope.com/",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/descope/core-js-sdk.git"
+  },
+  "autoupdate": {
+    "source": "npm",
+    "target": "@descope/core-js-sdk",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": ["**/*"]
```

```suggestion
        "files": [
          "*.js?(.map)",
          "cjs/*.js?(.map)",
          "*.d.ts"
        ]
```

Please avoid using globstars, as it can lead to including files on the CDN that were not intended -- looking at your package structure, there are some js & ts files that can be included.